src/AtmosphereModels/atmosphere_model.jl

```diff
@@ -14,6 +15,7 @@ using Oceananigans.Utils: launch!, prettytime, prettykeys, with_tracers
 
 import Oceananigans: fields, prognostic_fields
 import Oceananigans.Advection: cell_advection_timescale
+import Oceananigans.Diagnostics: default_nan_checker
```

Comment:
`import` 😢

Reply:
it must be `import` since I'm extending the method, right?

Reply:
No, a better way to extend a function from another method is to qualify the function with the owning module:
```julia
Diagnostics.default_nan_checker(...) = ...
```
`import` is a bit evil because you may accidentally extend methods from other modules if don't realise you `import`ed the function before (it's a sort of spooky action at a distance).  With the `Module.function(...) = ...` syntax you make it super explicit, locally, that you're extending a function from another module.  In some cases `import` is actively harmful: https://github.com/CliMA/Oceananigans.jl/pull/5041#issuecomment-3639653393.  Just completely avoid `import` to extend methods from other modules and live happy.

Reply:
see https://github.com/NumericalEarth/Breeze.jl/pull/294